chall02/rgallard.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+morse = {
+	'A':'.-',
+	'B':'-...',
+	'C':'-.-.',
+	'D':'-..',
+	'E':'.',
+	'F':'..-.',
+	'G':'--.',
+	'H':'....',
+	'I':'..',
+	'J':'.---',
+	'K':'-.-',
+	'L':'.-..',
+	'M':'--',
+	'N':'-.',
+	'O':'---',
+	'P':'.--.',
+	'Q':'--.-',
+	'R':'.-.',
+	'S':'...',
+	'T':'-',
+	'U':'..-',
+	'V':'...-',
+	'W':'.--',
+	'X':'-..-',
+	'Y':'-.--',
+	'Z':'--..',
+	' ':' '
+	}
+
+i = 1
+j = 0
+talk = ""
+
+if (2 > len(sys.argv)):
+	talk = "<a-zA-Z string>"
+while (i < len(sys.argv)):
+	for j in sys.argv[i]:
+		j = j.upper()
+		if (j in morse.keys()):
```

Comment:
Good 👍 it's a better way to check if it's a writable character than use the functions `isascii` or `isalpha` but it makes you declare a variable in order to store your result when you could write if you've done the checks before